features/support/pages/app.rb

```diff
@@ -111,4 +111,17 @@ def public_project?(project: nil, title: nil)
     current_page.project?(title: title)
   end
   alias has_public_project? public_project?
+
+  def emails_to(address)
+    ActionMailer::Base.deliveries.select do |email|
+      email.to.include?(address)
+    end.map(&TestEmail.method(:new))
```

Comment:
We may want to pull the mapping to the wrapper to a higher level, but this seemed OK for now.

Reply:
Also @bhaibel or @jtu0 - do you know of a gem that gives us this kind of stuff for free without infecting our project with a billion step definitions?